samtranslator/model/connector/connector.py

```diff
@@ -89,6 +89,17 @@ def _is_valid_resource_reference(obj: Dict[str, Any]) -> bool:
     return id_provided != non_id_provided
 
 
+def get_underlying_cfn_resource_type(resource_type: str) -> str:
+    sam_to_cfn_types = {
+        "AWS::Serverless::Function": "AWS::Lambda::Function",
+        "AWS::Serverless::StateMachine": "AWS::StepFunctions::StateMachine",
+        "AWS::Serverless::Api": "AWS::ApiGateway::RestApi",
+        "AWS::Serverless::SimpleTable": "AWS::DynamoDB::Table",
+        "AWS::Serverless::HttpApi": "AWS::ApiGatewayV2::Api",
+    }
```

Comment:
this is a constant, it should be at the module level

Reply:
Can we avoid re-typing them to avoid typos? Like `LambdaFunction.resource_type` and `SamResourceType.Function`.

Reply:
Good suggestion!